modules/artifact/src/main/java/org/mule/module/artifact/classloader/CompositeClassLoader.java

```diff
@@ -32,45 +36,85 @@
     protected static final Log logger = LogFactory.getLog(CompositeClassLoader.class);
 
     protected final List<ClassLoader> classLoaders;
-
-    public CompositeClassLoader(ClassLoader parent, List<ClassLoader> classLoaders)
+    private final ClassLoaderLookupPolicy lookupPolicy;
+
+    /**
```

Comment:
Update class javadoc to explain how with CL uses lookup policy rather than simply stating that it delegates to a list of CL's.